lib/oca-epak/epak/client.rb

```diff
@@ -61,23 +45,26 @@ def create_pickup_order(opts = {})
       # Get rates and delivery estimate for a shipment
       #
       # @param [Hash] opts
-      # @option [String] :wt Total Weight e.g: 20
-      # @option [String] :vol Total Volume e.g: 0.0015 (0.1mts * 0.15mts * 0.1mts)
-      # @option [Integer] :origin Origin ZIP Code
-      # @option [Integer] :destination Destination ZIP Code
-      # @option [Integer] :qty Quantity of Packages
-      # @option [Integer] :total Declared Value
+      # @option [String] :total_weight Total Weight e.g: 20
+      # @option [String] :total_volume Total Volume e.g: 0.0015
+      #                                (0.1mts * 0.15mts * 0.1mts)
+      # @option [String] :origin_zip_code Origin ZIP Code
+      # @option [String] :destination_zip_code Destination ZIP Code
+      # @option [String] :declared_value Declared Value
+      # @option [String] :package_quantity Quantity of Packages
       # @option [String] :cuit Client's CUIT e.g: 30-99999999-7
-      # @option [String] :op Operation Type
-      # @return [Hash, nil] Contains Total Price, Delivery Estimate
-      def get_shipping_rates(opts = {})
+      # @option [String] :operation_code Operation Type
+      # @return [Array, nil] Contains Total Price, Delivery Estimate
```

Comment:
@etagwerker hmm, I'm not sure, I may have to add some coverage here. I think `BaseClient#parse_results_table` could return `nil`. I'll look into it.